### PR TITLE
feat: Passing Resources Between Stacks

### PR DESCRIPTION
**Pull request by richardhboyd**
Added a Python Example for passing resources between stacks

---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

File: python/cross-stack-resources/native-objects/native_objects/infrastructure_stack.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+        my_main_func = lambda_.Function(
+            self,
+            "myMainFunction",
+            code=lambda_.InlineCode("def main(event, context)\n  print('hello, world')"),
+            handler='index.main',
+            runtime=lambda_.Runtime.PYTHON_3_7
+        )
+        
+        # We assign the function's arn to a local variable for the Object.
```
Comment (richardhboyd) [inline python/cross-stack-resources/native-objects/native_objects/infrastructure_stack.py]: Change this comment, it is an Object, not an ARN that is being assigned.

Comment (NGL321) [reply python/cross-stack-resources/native-objects/native_objects/infrastructure_stack.py]: Is this a self review? Should we wait for this change?

File: python/cross-stack-resources/raw-strings/setup.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import setuptools
+
+
+with open("README.md") as fp:
+    long_description = fp.read()
+
+
+setuptools.setup(
+    name="raw_strings",
+    version="0.0.1",
```
Comment (NGL321) [inline python/cross-stack-resources/raw-strings/setup.py]: Set version to 1.0.0

File: python/cross-stack-resources/raw-strings/setup.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+    description="A sample CDK Python app",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+
+    author="author",
```
Comment (NGL321) [inline python/cross-stack-resources/raw-strings/setup.py]: Set name and description

File: python/cross-stack-resources/native-objects/setup.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import setuptools
+
+
+with open("README.md") as fp:
+    long_description = fp.read()
+
+
+setuptools.setup(
+    name="native-objects",
+    version="0.0.1",
```
Comment (NGL321) [inline python/cross-stack-resources/native-objects/setup.py]: Version to 1.0.0

File: python/cross-stack-resources/native-objects/setup.py [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+    description="A sample CDK Python app",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+
+    author="author",
```
Comment (NGL321) [inline python/cross-stack-resources/native-objects/setup.py]: Set author and description values

**Pull request merged**